sys/newlib_syscalls_default/syscalls.c

```diff
@@ -516,3 +535,17 @@ int _gettimeofday_r(struct _reent *r, struct timeval *restrict tp, void *restric
     return -1;
 }
 #endif
+
+/**
+ * Current process times (not implemented).
+ *
+ * @param[out]  ptms    Filled with zeros.
+ *
+ * @return  Zero.
+ */
+clock_t _times(struct tms *ptms)
+{
+    memset(ptms, 0, sizeof(*ptms));
+
```

Comment:
does errno have to be cleared? Spec says, errno indicates error on error...

Reply:
@kaspar030 You are right, this is an example from libgloss (part of newlib):
```
/* _times -- no clock, so return an error.  */
clock_t _times (struct tms *buf)
{
  errno = EINVAL;
  return (-1);
}
```
I'm fixing it.